index.html

```diff
@@ -1024,7 +1024,8 @@ <h3>Contexts</h3>
           <dd>
 The value of the <code>@context</code> <a>property</a> MUST be an ordered set
 where the first item is a <a>URI</a> with the value
-<code>https://www.w3.org/2018/credentials/v1</code>. Subsequent items in the
+<code>https://www.w3.org/2018/credentials/v1</code>. A copy of the base context is
+provided for refrence in the appendix <a href="#base-context"></a>. Subsequent items in the
 array MUST be composed of any combination of <a>URIs</a> or objects that
 express context information. It is RECOMMENDED that each <a>URI</a> in the
```

Comment:
```
provided for refrence in the appendix <a href="#base-context"></a>. Subsequent items in the
array MUST be composed of any combination of <a>URIs</a> or objects that
express context information.
```
->
```
provided for reference in the appendix <a href="#base-context"></a>. Subsequent items in the
array MUST be composed of a combination of <a>URIs</a> and/or objects that
expresses context information.
```
* `refrence` -> `reference`
* `any combination` -> `a combination`
* `or` -> `and/or`
* `express` -> `expresses`

Reply:
The typo I introduced was fixed, although the next 3 bullets where issues present prior to my PR.  While I don't disagree, those changes should be addressed separately I think (unless I missed something).

Reply:
Fair enough. (Better not to mark conversation as resolved if there's a pending action, such as my new PR.)